lib/libzpool/uzfs_mgmt.c

```diff
@@ -447,6 +451,8 @@ uzfs_close_dataset(zvol_state_t *zv)
 	zil_close(zv->zv_zilog);
 	dnode_rele(zv->zv_dn, zv);
 	dmu_objset_disown(zv->zv_objset, zv);
+	mutex_destroy(&zv->io_tree_mtx);
+	uzfs_destroy_txg_diff_tree(zv->incoming_io_tree);
```

Comment:
Same comment goes here too, we can remove/destroy when we done with Rebuild.

Reply:
yes. We can shift this operations to somewhere during integration or complete flow. As of now, i have kept this operation here just to make it complete flow without replica/target. I will add comments here.